### PR TITLE
Document ec2_key module return values

### PR DESCRIPTION
**Pull request by lorin**
##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
ec2_key module

##### ANSIBLE VERSION

```
ansible 2.2.1.0
  config file = 
  configured module search path = Default w/o overrides
```

##### SUMMARY
Document return values for ec2_key module

### CODE REVIEW

File: lib/ansible/modules/cloud/amazon/ec2_key.py [hunk 1]
```diff
@@ -95,6 +95,25 @@
     state: absent
 '''
 
+RETURN = '''
+key.fingerprint:
```
Comment (bcoca) [inline lib/ansible/modules/cloud/amazon/ec2_key.py:99]: the 'key' key itself should be part of the docs, type complex and then you can show the 'subkeys' as part of it

Comment (lorin) [reply lib/ansible/modules/cloud/amazon/ec2_key.py:99]: @bcoca It appears that the subkeys of complex types currently aren't rendered by the docs (e.g.,  [docker_service][1], [os_port_facts][2]). I'm loathe to make a change that means this info won't be visible in the docs. Should I submit a separate ticket for the issue of subkeys not being rendered? 

[1]: https://docs.ansible.com/ansible/docker_service_module.html#return-values
[2]: https://docs.ansible.com/ansible/os_port_facts_module.html#return-values

Comment (lorin) [reply lib/ansible/modules/cloud/amazon/ec2_key.py:99]: I opened #21556

Comment (s-hertel) [reply lib/ansible/modules/cloud/amazon/ec2_key.py:99]: Complex return docs now are generated.

**Pull request closed without merging**